examples/echobot.py

```diff
@@ -14,4 +16,21 @@ async def main():
                 await event.thread.send_text(event.message.text)
 
 
+async def main():
+    session = await fbchat.Session.login("<email>", "<password>")
+
+    client = fbchat.Client(session=session)
+    listener = fbchat.Listener(session=session, chat_on=False, foreground=False)
+
+    listen_task = asyncio.create_task(listen(listener, session))
+
+    client.sequence_id_callback = listener.set_sequence_id
+
+    # Fetch the last 10 threads that have had activity
+    await client.fetch_threads(limit=10).__anext__()
```

Comment:
```suggestion
    # Call the fetch_threads API once to get the latest sequence ID
    await client.fetch_threads().__anext__()
```

Reply:
But there needs to be an argument, otherwise:
`TypeError: fetch_threads() missing 1 required positional argument: 'limit'`

Reply:
Oh right it's `Optional` but doesn't have a default value